src/forms/FormFieldAdvancedSelection/AdvancedSelection.styled.tsx

```diff
@@ -30,7 +30,7 @@ export const InputWrapper = styled.div`
   justify-content: space-between;
   margin-bottom: 40px;
   padding: 0 16px;
-  width: ${pr => pr.isMobileView ? "" : "650px"};
+  width: ${pr => pr.isMobileView ? "" : "684px"};

```

Comment:
Let's add an "align-items: center" here, that way the "Create" button will be vertically aligned in the field.